experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -351,7 +357,12 @@ function get_atmos_config_dict(coupler_dict::Dict, job_id::String)
     # merge configs
     # (if there are common keys, the last dictionary in the `merge` arguments takes precedence)
     atmos_config = merge(atmos_config, Dict("output_dir" => atmos_output_dir))
```

Comment:
```suggestion
    dt_atmos = haskey(coupler_dict, "dt_atmos") ? coupler_dict["dt_atmos"] : coupler_dict["dt"]
    atmos_config = merge(atmos_config, Dict("output_dir" => atmos_output_dir), "dt" => dt_atmos)
```
Instead of modifying the entry `coupler_dict["dt"]` above and below here, we can find whichever entry exists in `coupler_dict`, and add the `"dt"` entry to `atmos_config` as we create it.